src/ext/NetFx/wixext/NetFxCompiler.cs

```diff
@@ -325,5 +361,184 @@ private void ParseNativeImageElement(Intermediate intermediate, IntermediateSect
                 });
             }
         }
+
+        /// <summary>
+        /// Parses a DotNetCompatibilityCheck element.
+        /// </summary>
+        /// <param name="element">The element to parse.</param>
+        private void ParseDotNetCompatibilityCheckElement(Intermediate intermediate, IntermediateSection section, XElement element)
```

Comment:
Again, @rseanhall was deeper in the design of this feature. I think it'd be good if he evaluates the design.

Reply:
I agree, let's see what Sean has to say about the PR and then I'll make all the changes requested by both of you.